### ISSUE TITLE
PickleJS: Integration Testing in Plain English

### ISSUE THREAD

**Issue by tolicodes** [wafflejs/wafflejs.github.io#601]
Having a full suite of tests is the key to getting to Continuous Delivery, allowing developers to safely experiment and push features without risking major bugs. But who actually likes writing tests? It’s so manual, mundane, and tedious.

PickleJS makes testing easy and fun. You write your tests in plain flexible English syntax (ex: I click on the “Checkout Button”)! Even PMs can do it. And it works with Cypress, Selenium, and can easily be adapted to whatever the next latest and greatest testing framework is, without changing your tests. As a bonus, your tests now double as documentation of your product that anyone can read.

PickleJS is used all throughout HOVER and is gaining popularity in the Open Source community, being actively used and developed in startups and corporations.

Check out the presentation on PickleJS.com for an idea of what Pickle can do.
Benefits
- Makes testing super simple (even non developers can do it)
- Eliminates duplication by abstracting common scenarios (clicking, typing, etc)
- Very simple to extend. The framework is layered so that you can inject plugins and functionality at any level with ease (ex: using our regex engine, creating custom phrases, building adapters for testing framework)
- Very simple and quick adoption, Junior developers love it!!!
- Portability of tests (same test works on Cypress and Selenium, with or without API mocking)
- Tests double as documentation

**Comment by andytuba** [comment]
Hi @tolicodes! Sorry for the delay, I've sent this into the WaffleJS organizers for review.
